internal/pkg/cli/app_delete.go

```diff
@@ -174,14 +148,6 @@ func (o *deleteAppOpts) sourceProjectEnvironments() error {
 		return fmt.Errorf("get environments: %w", err)
 	}
 
-	if len(envs) == 0 {
```

Comment:
Suppose you run `app init` and you have ssm store params and local files and you want to delete them. If you have to have at least one env then it will fail. Plus it does not make sense to error out actually: if there's no env then just don't delete application in any env.